### ISSUE TITLE
`track_corrections` function does not update the config file properly

### ISSUE THREAD

**Issue by valosekj** [spinalcordtoolbox/manual-correction#66]
The [`track_corrections` function ](https://github.com/spinalcordtoolbox/manual-correction/blob/6027fac2b4653c17c5ca7060244cdf4f24947961/utils.py#L298-L321) does not update the config file properly.

In the example below, I corrected the labels for subjects 008, 009, 010, and 011. All four subjects have been moved correctly under the `CORR_LESION` key, but 008, 009, and 010 were also kept under the `FILES_LESION` key. See the yml below:

```yml
CORR_LESION:
- sub-ott001_acq-sag_T2w.nii.gz
- sub-ott002_acq-sag_T2w.nii.gz
- sub-ott003_acq-sag_T2w.nii.gz
- sub-ott004_acq-sag_run-01_T2w.nii.gz
- sub-ott005_acq-sag_run-01_T2w.nii.gz
- sub-ott006_acq-sag_T2w.nii.gz
- sub-ott007_acq-sag_T2w.nii.gz
- sub-ott008_acq-sag_T2w.nii.gz
- sub-ott009_acq-sag_T2w.nii.gz
- sub-ott010_acq-sag_T2w.nii.gz
- sub-ott011_acq-sag_run-03_T2w.nii.gz
FILES_LESION:
- sub-ott008_acq-sag_T2w.nii.gz
- sub-ott009_acq-sag_T2w.nii.gz
- sub-ott010_acq-sag_T2w.nii.gz
```

@NathanMolinier haven't you encountered such behaviour?

**Comment by NathanMolinier** [comment]
This is actually really strange, the script was working perfectly for all the images but not for these 3. Did you check if no additional spaces are present at the end of the filenames ?

**Comment by valosekj** [comment]
> This is actually really strange, the script was working perfectly for all the images but not for these 3.

Sorry, I should have mentioned that I started the script only for the last 3 subjects (i.e., from `sub-ott008`).

**Comment by valosekj** [comment]
Fixed in #68
